consensus/rules/src/decomposed_amount.rs

```diff
@@ -1,13 +1,9 @@
-use std::sync::OnceLock;
-
-/// Decomposed amount table.
-///
-static DECOMPOSED_AMOUNTS: OnceLock<[u64; 172]> = OnceLock::new();
+use std::sync::LazyLock;
 
 #[rustfmt::skip]
-pub fn decomposed_amounts() -> &'static [u64; 172] {
-    DECOMPOSED_AMOUNTS.get_or_init(|| {
-        [
+/// Decomposed amount table.
+pub static DECOMPOSED_AMOUNTS: LazyLock<[u64; 172]> = LazyLock::new(|| {
```

Comment:
Can we remove the `LazyLock` here, keep it a static though. I don't know why I made this a `OnceLock`, the data is still in the binary.

Reply:
I must say I'm confused on why this isn't a `const`

Reply:
`static` is a ref to 1 memory address containing the data, `const` inlines the data everywhere it gets used.

I assume @Boog900 doesn't want it inlined in the binary everywhere it gets used.

Reply:
I was confused, I thought const weren't inlined. Thanks for the heads-up